lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -350,22 +361,17 @@ private Map<String, Object> buildNoMatchResponse(
         return JOURNEY_NEXT;
     }
 
-    private Map<String, Object> buildProfileMatchedResponse(
-            Gpg45Profile matchedProfile, AuditEventUser auditEventUser) throws SqsException {
-        var message =
-                new StringMapMessage()
-                        .with(
-                                LOG_MESSAGE_DESCRIPTION.getFieldName(),
-                                "Matched profile and within CI threshold so returning reuse journey.")
-                        .with(LOG_PROFILE.getFieldName(), matchedProfile.getLabel());
-        LOGGER.info(message);
-
-        auditService.sendAuditEvent(
-                new AuditEvent(
-                        AuditEventTypes.IPV_IDENTITY_REUSE_COMPLETE,
-                        configService.getSsmParameter(ConfigurationVariable.COMPONENT_ID),
-                        auditEventUser));
+    private Map<String, Object> buildReuseResponse(Vot vot, AuditEventUser auditEventUser)
+            throws SqsException {
+        if (vot.requiresProfileType(GPG_45)) {
+            auditService.sendAuditEvent(
+                    new AuditEvent(
+                            AuditEventTypes.IPV_IDENTITY_REUSE_COMPLETE,
+                            configService.getSsmParameter(ConfigurationVariable.COMPONENT_ID),
+                            auditEventUser));
+        }
```

Comment:
I've kept this audit event only for P2 vots in the assumption we're going to need a different audit event for inherited identity reuse journeys.